OpenRA.Mods.Common/Pathfinder/PathSearch.cs

```diff
@@ -46,13 +50,19 @@ public static IPathSearch FromPoint(IWorld world, IMobileInfo mi, IActor self, C
 				heuristic = DefaultEstimator(target)
 			};
 
+			search.isGoal = loc =>
+			{
+				var locInfo = search.Graph[loc];
+				return locInfo.EstimatedTotal - locInfo.CostSoFar == 0;
+			};
```

Comment:
It's just a factory constructor so it's not a big deal. Besides, from a behavioural and reusability aspect, the isGoal delegate should only know about location as a parameter (for searches and stuff like that). That delegate could be perfectly placed as a private method as it was before, but I consider it pointless).